src/redux/features/participants/participantsSlice.ts

```diff
@@ -1,21 +1,75 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 import { StatusT, buildReducers } from 'src/redux/features/asyncUtils';
-import { requestParticipantsList } from 'src/redux/features/participants/participantsAsyncThunks';
-import { Participant } from 'src/types/Participant';
+import { ParticipantData, ParticipantsData } from 'src/services/ParticipantsService';
+
+import {
+  requestDeleteParticipant,
+  requestGetParticipant,
+  requestListParticipants,
+  requestUpdateParticipant,
+} from './participantsAsyncThunks';
+
+/**
+ * Delete Participant
+ */
+const initialStateParticipantDelete: { [key: string]: StatusT<number> } = {};
+
+export const participantDeleteSlice = createSlice({
+  name: 'participantDeleteSlice',
+  initialState: initialStateParticipantDelete,
+  reducers: {},
+  extraReducers: (builder) => {
+    buildReducers(requestDeleteParticipant)(builder);
+  },
+});
+
+export const participantDeleteReducer = participantDeleteSlice.reducer;
+
+/**
+ * Get Participant
+ */
+const initialStateParticipant: { [key: string]: StatusT<ParticipantData> } = {};
+
+export const participantSlice = createSlice({
+  name: 'participantSlice',
+  initialState: initialStateParticipant,
+  reducers: {},
+  extraReducers: (builder) => {
+    buildReducers(requestGetParticipant, true)(builder);
+  },
+});
+
+export const participantReducer = participantSlice.reducer;
 
 /**
  * Participant list
  */
-const initialStateParticipantsList: { [key: string]: StatusT<Participant[]> } = {};
+const initialStateParticipantList: { [key: string]: StatusT<ParticipantsData> } = {};
+
+export const participantListSlice = createSlice({
+  name: 'participantListSlice',
+  initialState: initialStateParticipantList,
+  reducers: {},
+  extraReducers: (builder) => {
+    buildReducers(requestListParticipants)(builder);
+  },
+});
+
+export const participantListReducer = participantListSlice.reducer;
+
+/**
+ * Participant update
+ */
+const initialStateParticipantUpdate: { [key: string]: StatusT<number> } = {};
 
-export const participantsListSlice = createSlice({
-  name: 'participantsListSlice',
-  initialState: initialStateParticipantsList,
+export const participantUpdateSlice = createSlice({
```

Comment:
I have been merging delete/update as `xyzEditSlice` since it captures a requestId/object-id state. Having them separate is fine too, just in case you prefer to merge them into one.